### PR TITLE
Fixed link in tune that was not working

### PR DESCRIPTION
**Pull request by lukasfolle**
## What do these changes do?
Fixed link leading to 404 page.

### CODE REVIEW

**Review by richardliaw — APPROVED** [review]

File: python/ray/tune/README.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ User documentation can be `found here <http://ray.readthedocs.io/en/latest/tune.
 Tutorial
 --------
 
-To get started with Tune, try going through `our tutorial of using Tune with Keras <https://github.com/ray-project/tutorial/blob/master/tune_exercises/Tune.ipynb>`__.
+To get started with Tune, try going through `our tutorial of using Tune with Keras <https://github.com/ray-project/ray/blob/master/python/ray/tune/examples/tune_mnist_keras.py>`__.
```
Comment (richardliaw) [inline python/ray/tune/README.rst]:
```suggestion
To get started with Tune, try going through `our tutorial of using Tune with Keras <https://github.com/ray-project/tutorial/blob/master/tune_exercises/exercise_1_basics.ipynb>`__.
```

**Review by richardliaw — CHANGES_REQUESTED** [review]
Thanks for the fix! I think this is pointing to the wrong tutorial intended still.

**Comment by lufol** [comment]
You are right!

**Review by richardliaw — APPROVED** [review]

**Comment by lufol** [comment]
The error still persists on the doc page https://ray.readthedocs.io/en/latest/tune.html
Can someone also correct it there?

**Pull request merged**